concepts/05 UI Components/TabPanel/01 Getting Started with TabPanel/20 Create Tabs.md

```diff
@@ -0,0 +1,105 @@
+To create tabs, specify the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array. TabPanel generates a tab with a view for each item in this array. 
```

Comment:
```suggestion
Populate the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array with data items. The TabPanel generates a tab with a view for each item in this array.
```

Reply:
Shouldn't we say about the purpose of this action first? How about the following:
To create tabs, populate the [items[]](/Documentation/ApiReference/UI_Components/dxTabPanel/Configuration/items/) array with data. The TabPanel generates...